body_force_estimator/src/body_force_estimator_node.cpp

```diff
@@ -0,0 +1,12 @@
+#include <ros/ros.h>
+#include "body_force_estimator/body_force_estimator.h"
+
+int main(int argc, char** argv) {
+	ros::init(argc, argv, "body_force_estimator_node");
+	ros::NodeHandle nh;
+
+	BodyForceEstimator body_force_estimator(nh);
+	body_force_estimator.spin();
+
+	return 0;
+}
```

Comment:
Fix indentation (should be 2 spaces)

Reply:
(We also need to fix it in other places but might as well maintain the standard until we do)

Reply:
Indentation fixed in this and other files.